symengine/lib/symengine_wrapper.pyx

```diff
@@ -295,6 +295,8 @@ def sympy2symengine(a, raise_error=False):
                 return RealDouble(float(str(a)))
         ELSE:
             return RealDouble(float(str(a)))
+    elif have_numpy and isinstance(a, (np.float16, np.float32)):
+        return RealDouble(a)
```

Comment:
Delete these lines. The other two are sufficient

Reply:
Done in: https://github.com/symengine/symengine.py/pull/352/commits/97ded0836bcaa81d2b527184d75598587c52d7d0